### ISSUE TITLE
semantic-release-slack-bot@2.1.2 requires Node 14 which seems to be a breaking change

### ISSUE THREAD

**Issue by bensampaio** [juliuscc/semantic-release-slack-bot#78]
I am working on a project that requires Node 12 and when upgrading dependencies with patch and minor releases I got this warning:

```
error semantic-release-slack-bot@2.1.2: The engine "node" is incompatible with this module. Expected version ">=14.7". Got "12.21.0"
```

Shouldn't such a change be released as a major release?

**Comment by juliuscc** [comment]
Yes, that is a good point. That is a miss that we didn't trigger a breaking change. Even though we publish a new breaking change the published version will stay. What do you think we should do?

Would it solve your issue if we deprecate the latest published version and then publish a new version under a new major version?

**Comment by bensampaio** [comment]
I think a new major release would be a good approach. You could deprecate 2.1.2 and add to the release notes that there was an unintended breaking change. Then we wouldn't need to fix the version to 2.1.1 and others will know that there are breaking changes.

**Comment by juliuscc** [comment]
I have now deprecated version `2.1.2` and published a new major version (`3.0.0`).

Thank you for being patient with us, and please tell us if it solves your issue or not!

**Comment by bensampaio** [comment]
Thank you @juliuscc! That went quite fast, this should indeed fix our problem :)
